src/inventory/disks.go

```diff
@@ -55,8 +55,12 @@ func (d *disks) getDisks() []*models.Disk {
 		return ret
 	}
 	for _, disk := range blockInfo.Disks {
-		if disk.IsRemovable || disk.BusPath == ghw.UNKNOWN {
+		if disk.IsRemovable {
```

Comment:
Maybe unify all conditions to one single OR condition - no need for else

Reply:
> Maybe unify all conditions to one single OR condition - no need for else

Done.